shared/actions/chat.js

```diff
@@ -72,7 +73,10 @@ const {
 
 const {conversationIDToKey, keyToConversationID, InboxStateRecord, MetaDataRecord, makeSnippet, serverMessageToMessageBody} = Constants
 
-const _selectedSelector = (state: TypedState) => state.chat.get('selectedConversation')
+const _selectedSelector = (state: TypedState) => {
+  const selected = getPath(state.routeTree.routeState, [chatTab]).last()
```

Comment:
per our discussion this make assumptions about the route that'll likely change (esp. w/ modals)